db/range_del_aggregator_bench.cc

```diff
@@ -52,6 +52,8 @@ DEFINE_int32(seed, 0, "random number generator seed");
 
 DEFINE_int32(should_deletes_per_run, 1, "number of ShouldDelete calls per run");
 
+DEFINE_int32(add_tombstones_per_run, 1, "number of AddTombstones calls per run");
```

Comment:
What exactly is the purpose of this parameter? Why not just set num_range_tombstones appropriately?

Reply:
Although there's currently not much difference between calling `AddTombstones` multiple times and calling `AddTombstones` once with a larger number of tombstones, in the new design, `RangeDelAggregator` is planned to be a wrapper around a heap of `InternalIterator`s. Then, `AddTombstones` multiple times will influence the performance of `ShouldDelete` in a different way than calling `AddTombstones` once.